tex/gregoriotex-main.tex

```diff
@@ -560,18 +620,25 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 % format for text above the lines
-\def\greabovelinestextstyle#1{%
-#1\relax %
+\def\greabovelinestextstyle#1{% DEPRECATED
+  \relax %
 }%
 
 % set space above the text lines - almost the same as for the translation
 \def\greaddspaceabove{%
-  \gre@style@abovelinestext%
-  \greabovelinestextstyle{%
-    \global\gre@dimen@currentabovelinestextheight=\gre@dimen@abovelinestextheight %
-    \gregeneratelines %
-  }%
-  \endgre@style@abovelinestext%
+  \protected@edef\gre@@arg{\greabovelinestextstyle{a}}%  DEPRECATED
+  \expandafter\ifx\gre@@arg\relax% DEPRECATED
```

Comment:
One of the adaptations I had to make was to use `\protected@edef` instead of just `\def` (which is what you used in the point-and-click application).  Given that the error you've noticed is related to `\protected@edef` I don't think it should be a problem.